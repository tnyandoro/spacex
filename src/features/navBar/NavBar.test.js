import React from 'react';
import { HashRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from './NavBar';

it('NavBar snapshot test', () => {
  const routes = [
    {
      name: 'Rockest',
      path: '/',
    },
  ];
  const tree = renderer
    .create(
      <Router>
        <NavBar title="Mock" routes={routes} />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
