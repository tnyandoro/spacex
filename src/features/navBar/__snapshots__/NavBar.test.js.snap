// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar snapshot test 1`] = `
<nav
  className="navBar"
>
  <ul
    className="nav-list"
  >
    <li
      className="brand"
    >
      <a
        className="nav-brand"
        href="#/"
        onClick={[Function]}
      >
        <img
          alt="logo"
          className="App-logo"
          src="planet.png"
        />
        <h1
          className="nav-title"
        >
          Mock
        </h1>
      </a>
    </li>
    <li
      className="nav-links"
    >
      <a
        aria-current="page"
        className="links active"
        exact="true"
        href="#/"
        onClick={[Function]}
      >
        Rockest
      </a>
    </li>
  </ul>
</nav>
`;
